### DISCUSSION TITLE
GPU Share Memory

### DISCUSSION THREAD

**Discussion by kdg5424** [lululxvi/deepxde#1320] [Q&A]
Hi~ I'm Mr. kim. I'm a CFD engineer.
And these days, I'm testing DeepXDE with some CFD examples. And I have a question.
I checked GPU Memory and usage with task manager while using DeepXDE. And I found that GPU used just GPU memory. GPU doesn't use share memory.
I'm using GeForce RTX 3060. The GPU memory is 8GB, although share memory 32GB.  My CUDA version is 12.1
I want to know how I can use GPU Share memory.
Could you answer my question?

Thank you!!
Best regards~

![image](https://github.com/lululxvi/deepxde/assets/133930861/349524d3-fea6-40f4-940c-19cb0889452b)

**Comment by lululxvi** [comment]
As I know, you cannot.

**Reply by kdg5424** [reply]
Thanks. Lu Lu.
